test/dynamo/test_bridge.py

```diff
@@ -266,6 +266,25 @@ def foo(device):
 
     self._compile_and_check(foo, (xm.xla_device(),))
 
+  def test_cpu_index_and_output(self):
+
+    def foo(xt, t):
+      # Cannot move t to XLA, since it's also an output.
+      return (xt * 5)[t], t
```

Comment:
I am not fully following what this test is trying to test. Can you make comment more explicit? For example without this change how/why will it fail?